docs/src/main/asciidoc/kubernetes.adoc

```diff
@@ -464,15 +464,15 @@ Below you will find tables describing all available types.
 |====
 | Property        | Type    | Description | Default Value
 | config-map-name | String  |             |
-| default-mode    | int     |             | 384
+| default-mode    | int     |             | 0600
 | optional        | boolean |             | false
 |====
 
 .SecretVolume
 |====
 | Property     | Type    | Description | Default Value
 | secret-name  | String  |             |
-| default-mode | int     |             | 384           
+| default-mode | int     |             | 0600
 | optional     | boolean |             | false
 |====
```

Comment:
Could someone really have a look to the generated documentation and see if we can use it?

If we have proper Quarkus config now, it will really be a big win.

Reply:
@iocanel didn't you say you were going to look into it?

Reply:
Maybe I will have a look and see how it goes.

Reply:
BTW, one things that bugs me is that we have things like `timeout-seconds` in the doc.

I don't know how they have been translated in Quarkus config but they should be `Duration` and named `timeout` without `-seconds`.

That's really something we want done before Final so either @iocanel can have a look or I will, just tell me.

Reply:
I will

Reply:
@iocanel feel free to ping me if you need some help or guidance on this one.